lib/orbit-common/local-storage-source.js

```diff
@@ -26,23 +26,25 @@ var LocalStorageSource = MemorySource.extend({
     this._super.apply(this, arguments);
 
     options = options || {};
-    this.namespace = options['namespace'] || 'orbit'; // local storage key
+    this.namespace = options['namespace'] || 'orbit'; // local storage namespace
+    this.delimeter = options['delimeter'] || '/'; // local storage key
     this._autosave = options['autosave'] !== undefined ? options['autosave'] : true;
     var autoload = options['autoload'] !== undefined ? options['autoload'] : true;
 
     this._isDirty = false;
 
-    this.on('didTransform', function() {
-      this._saveData();
-    }, this);
+    this.on('didTransform', this._saveData, this);
 
     if (autoload) this.load();
   },
 
   load: function() {
-    var storage = window.localStorage.getItem(this.namespace);
-    if (storage) {
-      this.reset(JSON.parse(storage));
+    for (var key in window.localStorage) {
+      if (key.startsWith(this.namespace)) {
```

Comment:
I agree with @ropsu here. We're not using babeljs (yet) so we need to be careful about compatibility.